### PR TITLE
feat: ✅ integrating language-page-design and next-i18-next

### PR DESCRIPTION
**Pull request by appsplash99**

### CODE REVIEW

**Review by anvaynk — CHANGES_REQUESTED** [review]

**Review by dewanshparashar — CHANGES_REQUESTED** [review]
Good PR. Small, minor comments.

File: package.json [hunk 1]
```diff
@@ -18,14 +18,15 @@
     "@emotion/css": "^11.7.1",
     "@emotion/react": "^11.8.2",
     "@emotion/styled": "^11.8.1",
+    "framer-motion": "^6.2.8",
```
[the remark this answers is not in the document]
Comment (appsplash99) [reply package.json:21]: is being used internally by chakra ui

File: .env.sample [hunk 2]
```diff
@@ -0,0 +1,5 @@
+NEXT_PUBLIC_AIRTABLE_BASE_ID=
+NEXT_PUBLIC_AIRTABLE_API_KEY=
+NEXT_PUBLIC_AIRTABLE_NEWS_BASE_NAME=
```
[the remark this answers is not in the document]
Comment (appsplash99) [reply .env.sample:3]: env.sample is public for open-source

File: package.json [hunk 3]
```diff
@@ -41,11 +42,11 @@
     "@types/react": "17.0.39",
     "eslint": "8.8.0",
     "eslint-config-next": "12.0.10",
-    "husky": "^7.0.0",
     "lint-staged": ">=10",
     "next-sitemap": "^2.5.20",
     "prettier": "2.6.1",
-    "typescript": "4.5.5"
+    "typescript": "4.5.5",
+    "husky": "^7.0.0"
```
[the remark this answers is not in the document]
Comment (appsplash99) [reply package.json:49]: already in devDeps

File: tsconfig.json [hunk 4]
```diff
@@ -16,6 +16,7 @@
     "incremental": true,
     "paths": {
       "@shm/Icons": ["components/common/Icons"],
+      "@shm/components/*": ["components/*"],
```
[the remark this answers is not in the document]
Comment (appsplash99) [reply tsconfig.json:19]: we can use this at later stage

File: public/locales/en/common.json [hunk 5]
```diff
@@ -0,0 +1,8 @@
+{
+  "site-title": "Welcome to Shardeum.org!",
+  "site-description": "Decentralization for everyone",
+  "language-en": "English",
```
[the remark this answers is not in the document]
Comment (appsplash99) [reply public/locales/en/common.json:2]: removed unwanted key-val pairs

**Review by dewanshparashar — APPROVED** [review]
LGTM

**Pull request merged**